apps/src/applab/applab.js

```diff
@@ -520,11 +520,11 @@ function selectEditorRowCol(row, col) {
     var range = selection.getRange();
 
     range.start.row = row;
-    range.start.col = col;
+    range.start.column = col;
     range.end.row = row;
-    range.end.col = col + 1;
+    range.end.column = col + 1;
 
-    selection.setSelectionRange(range);
+    selection.setSelectionRange(range, true);
```

Comment:
Reviewing for my own education!
This 'backwards' parameter is such an awesome thoughtful change.  It's a reason I wouldn't ever guess looking at this code, and would worry about some well-meaning person (like me) undoing later.  Will you comment this with the reason for the backwards=true change?